float64 x
float64 y
float64 z

float64 rotation_x
float64 rotation_y
float64 rotation_z
float64 rotation_w


float64 time
